lib/text/ui_text_displayer.js

```diff
@@ -382,15 +382,42 @@ shaka.text.UITextDisplayer = class {
       // TODO: Revisit this as we update the TTML parser.  See special case for
       // backgroundImage + region above.
       const percentageUnit = shaka.text.CueRegion.units.PERCENTAGE;
-      const heightUnit = cue.region.heightUnits == percentageUnit ? '%' : 'px';
-      const widthUnit = cue.region.widthUnits == percentageUnit ? '%' : 'px';
-      const viewportAnchorUnit =
-          cue.region.viewportAnchorUnits == percentageUnit ? '%' : 'px';
-      style.height = cue.region.height + heightUnit;
-      style.width = cue.region.width + widthUnit;
+
+      // Convert everything to percentages of the video container.
```

Comment:
This is so that we don't compare apples and oranges when we're doing the logic that makes use of region anchors (percentages), viewport anchors (percentages or pixels), and width/height (percentages or pixels).

Reply:
Makes sense to me.